packages/vue-components/src/Navbar.vue

```diff
@@ -198,6 +217,33 @@ export default {
         }
       }
     },
+    initSiteNav() {
+      this.hasSiteNav = true;
+      this.toggleNavMenu();
+    },
+    initPageNav() {
+      this.hasPageNav = true;
+      this.toggleNavMenu();
+    },
+    showNavMenu() {
+      this.isNavMenuShowing = true;
+    },
+    hideNavMenu() {
+      this.isNavMenuShowing = false;
+    },
+    toggleNavMenu() {
+      if ((this.hasPageNav && window.innerWidth < 1300) 
```

Comment:
how about (to avoid coupling this slot showing with the buttons):
- let the button components define their own `window.innerWidth` limits, and destroy themselves upon exceeding this limit  (or show themselves)
- let `navbar` do a generic check for whether there is any html within the div you added above (then this slot can also be used for non-button content)

Reply:
Shifted this to their individual component. `navbar` should be free of `page/site-nav-button` components other than the fallback content in the `slot`.